lunes_cms/static/js/color_unread_feedback.js

```diff
@@ -0,0 +1,19 @@
+/*
+ * Function to highlight unread feedback with bold letters
+*/
+
+
+function markUnreadFeedback () {
+    let readByEntries = document.querySelectorAll(".field-read_by");
+    if (readByEntries.length != 0) {
+        
+        readByEntries.forEach((entry)=>{
+            if (entry.textContent == "-") {
+                //entry.parentNode.style.backgroundColor = "yellow";
+                entry.parentNode.style.fontWeight = "bold";
```

Comment:
To me this change is still too subtle. Maybe @hauf-toni has some quick input for us on how we can highlight the feedback even a bit more. This is how it looks with Mizuki changes
![grafik](https://github.com/digitalfabrik/lunes-cms/assets/72705147/f7ed4780-6444-44e6-816e-988e69381d09) (the first line is read feedback, the second and third are unread.) Do you have any suggestions to make it stand out a bit more?

Reply:
Sorry, I wanted to color by `entry.parentNode.style.backgroundColor = "yellow";` which was somehow comment-outed 🙈

Reply:
I think I would be better to follow Toni's suggestion here. I think my suggested changes should do the trick. Then this PR should be good to go :)